public/data/shivam-sharma7.json

```
@@ -0,0 +1,28 @@
+{
+"name": " Shivam Sharma ",
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Fullstack Developer | DevOps Enthusiast",
+  "avatar": "https://github.com/shivam-sharma7.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shivam-sharma7",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/meshivamsharma",
+      "icon": "twitter"
+    },
+    {
+      "name": "Visit my profile website",
+      "url": "https://hashnode.com/@shivam-sharma",
```

Will better to land visitors directly to your work (Articles)

```suggestion
      "url": "https://shivam-sharma.in/",
```